### PR TITLE
Allow custom authenticationRoute / redirectUri

### PR DESCRIPTION
**Pull request by bjornharrtell**
Without this change setting the property `authenticationRoute` (from http://ember-simple-auth.com/api/classes/AuthenticatedRouteMixin.html) on has no effect.

### CODE REVIEW

File: addon/mixins/oidc-authentication-route-mixin.js [hunk 1]
```diff
@@ -21,7 +21,10 @@ export default Mixin.create(UnauthenticatedRouteMixin, {
 
   redirectUri: computed(function() {
     let { protocol, host } = location;
-    let path = this.router.urlFor(Configuration.authenticationRoute);
+    let route = this.authenticationRoute;
+    if (!route)
```
Comment (Yelinz) [inline addon/mixins/oidc-authentication-route-mixin.js]: Please use braces for the if statement

File: addon/mixins/oidc-authentication-route-mixin.js [hunk 2]
```diff
@@ -21,7 +21,11 @@ export default Mixin.create(UnauthenticatedRouteMixin, {
 
   redirectUri: computed(function() {
     let { protocol, host } = location;
-    let path = this.router.urlFor(Configuration.authenticationRoute);
+    let route = this.authenticationRoute;
```
Comment (Yelinz) [inline addon/mixins/oidc-authentication-route-mixin.js]: You could simplify this by doing `this.router.urlFor(this.authenticationRoute || Configuration.authenticationRoute)`

Comment (Yelinz) [reply addon/mixins/oidc-authentication-route-mixin.js]: The Tests are still failing because of eslint, please use prettier or eslint to fix it.

**Comment by bjornharrtell** [comment]
Thanks for merging and the patience @Yelinz. :)

**Comment by Yelinz** [comment]
No problem!

**Pull request merged**